lib/tabbed-dialog.jsx

```diff
@@ -30,8 +30,8 @@ export default React.createClass( {
 			<Dialog className="settings" {...dialog}>
```

Comment:
Yeah I wasn't sure how far to track this down. I believe that the problem existed before this branch, and it should be getting the key from that `dialog` prop. I can continue to dig in before we merge. Thanks!